yolox_torchscript.py

```diff
@@ -0,0 +1,13 @@
+import torch
+import torch.nn as nn
+from yolox.models.yolox import YOLOX
+
+
+class DeployModule(nn.Module):
+    def __init__(self):
+        super().__init__()
+        self.yolox = YOLOX()
```

Comment:
this class is designed to prepare deploy module version, so IMO it would be more intuitive to expect model in init parameters.